libs/application/template-api-modules/src/lib/modules/templates/old-age-pension/old-age-pension.service.ts

```diff
@@ -9,24 +13,39 @@ import { BaseTemplateApiService } from '../../base-template-api.service'
 import { TemplateApiModuleActionProps } from '@island.is/application/template-api-modules'
```

Comment:
Breyta import til að fjarlægja eslint villu
```suggestion
import { TemplateApiModuleActionProps } from '../../../types'
```